__tests__/app.test.js

```diff
@@ -93,4 +94,31 @@ describe('GET /api/articles/:article_id', () => {
                 expect(body.msg).toBe('Invalid article ID');
             });
     });
+});
+
+describe('GET /api/articles', () => {
+    test('responds with status 200 and an array of articles sorted by date', () => {
```

Comment:
If I was being _very_ picky I'd say the wording could be improved to directly refer to `created_at` to be a slightly better description of the behaviour this test is checking.